docs/pages/management/admin/trustedclusters.mdx

```diff
@@ -1,500 +1,594 @@
+  the Teleport Auth Service on the root cluster signs the certificate.
+- The **metadata certificate extensions** that are provided by the Teleport Auth Service. Teleport uses 
+  the metadata to store the list of user roles and SSH options like `permit-agent-forwarding`.
+- The expiration date, or **time-to-live** (TTL) to ensure the certificate is still valid.
+
+Based on the information in the certificate, the Teleport Auth Service performs the following actions:
+
+- Verifies that the certificate signature matches one of its trusted clusters.
+- Applies role mapping to associate a role on the leaf cluster with one of the roles assigned 
+  to the user in the root cluster.
+- Checks if the local role allows the requested identity—the Unix login—to have access.
+- Verifies that the certificate has not expired. The TTL is set by the root cluster.
+
+Note that trusted clusters only work in one direction. 
+Users from the leaf cluster can't see or connect to servers in the root cluster.
+
+## Role relationships in a trusted cluster
+
+As noted in [How it works](#how-it-works), leaf clusters are autonomous. They have their own state, roles, 
+and local users. This autonomy enables leaf cluster administrators to decide how to map the identity of the 
+external users to their local cluster roles. 
+The following diagram provides a simplified view of how role mapping works using the same trusted
+cluster with `sso.example.com` as the root cluster and `us-east-1a` as the leaf cluster:
+
+![Role mapping in a trusted cluster](../../../img/trusted-clusters/simplified-trusted-cluster-role-mapping.png)
+
+In this example, the user Alice signs in to the `sso.example.com` root cluster. The root cluster is
+configured with a single sign-on identity provider that authenticates her identity and her group membership.
+Based on the information from the identity provider, the root cluster assigns Alice the `full-access` role
+and issues her a certificate. The mapping of single sign-on properties to Teleport roles is configured when
+you add an authentication connector to the Teleport cluster. To learn more about configuring single sign-on
+through an external identity provider, see [Configure Single Sign-on](../../access-controls/sso).
+
+Alice receives the certificate that specifies the roles assigned to her in the root cluster. This metadata
+about her roles is contained in the certificate extensions and is protected by the signature of the root 
+cluster certificate authority so it can't be tampered with.
+
+When Alice connects to a resource in the leaf cluster `us-east-1a`, she's identified as an external user
+with a certificate signed by an external certificate authority. Based on the role mapping rules in the leaf
+cluster, Alice is assigned the `stage-access` role. This role allows her to access `mongodb.stage.us-east-1a`
+but not `mongodb.prod.us-east-1a`.
```

Comment:
Might be cool to illustrate role mappings as a table (one column for "role in the root cluster" and another column for "mapped role in the leaf cluster)